crates/but/src/tui/get_text.rs

```diff
@@ -60,13 +61,40 @@ fn is_terminal_dumb() -> bool {
 /// - Fall back to platform defaults (vi on Unix, notepad on Windows)
 /// - Add comprehensive unit tests for all scenarios
 fn get_editor_command() -> Result<String> {
-    get_editor_command_impl(&|key| std::env::var(key), is_terminal_dumb())
+    get_editor_command_impl(
+        &|key| std::env::var(key),
+        &|command| which(command).is_ok(),
+        is_terminal_dumb(),
+    )
+}
+
+/// Get the platform fallback editor.
+fn get_platform_editor<F>(command_exists: &F) -> String
+where
+    F: Fn(&str) -> bool,
+{
+    if cfg!(windows) {
+        "notepad".to_owned()
+    } else if cfg!(target_os = "linux") && command_exists("editor") {
+        // Linux: git source code has "vi" as fallback editor, but debian-based distros override this to
```

Comment:
I wonder, perhaps this comment is nicer as a function documentation, this way it gets surfaced in IDEs on hover etc